source/acquaman/AMScanConfiguration.h

```diff
@@ -277,6 +293,10 @@ public slots:
 	  * with the axes of the scan.
 	  */
 	void setAxisControlInfos(const AMControlInfoList &axisControlInfos);
+	/// Sets the automatic direction assessment flag.
+	void setAutomaticDirectionAssessment(bool isAutomatic);
+	/// Sets the direction.
+	void setDirection(Direction newDirection);
```

Comment:
Oh I see. It looks like there's a type mismatch for the direction argument here. It's different than what's listed in the .cpp file.